src/menus/tableRow.tsx

```diff
@@ -1,23 +1,25 @@
 import { EditorState } from "prosemirror-state";
 import { TrashIcon, InsertAboveIcon, InsertBelowIcon } from "outline-icons";
 import { MenuItem } from "../types";
+import baseDictionary from "../dictionary";
 
 export default function tableRowMenuItems(
   state: EditorState,
-  index: number
+  index: number,
+  dictionary: typeof baseDictionary
 ): MenuItem[] {
   return [
     {
-      name: "addRowAfter",
-      tooltip: "Insert row before",
+      name: "addRowBefore",
```

Comment:
I thought maybe this one was a mistake, so I fixed it. Please let me know if my assumption was wrong.

Reply:
It's actually not a mistake, the `attrs` make it behave differently

Reply:
Thank you, reverted.